packages/core/src/routes/swagger/utils/operation-id.ts

```diff
@@ -0,0 +1,196 @@
+import camelcase from 'camelcase';
+import { OpenAPIV3 } from 'openapi-types';
+import pluralize from 'pluralize';
+
+import { EnvSet } from '#src/env-set/index.js';
+
+import { shouldThrow } from './general.js';
+
+const chunk = <T>(array: T[], chunkSize: number): T[][] =>
+  Array.from({ length: Math.ceil(array.length / chunkSize) }, (_, i) =>
+    array.slice(i * chunkSize, i * chunkSize + chunkSize)
+  );
+
+const methodToVerb = Object.freeze({
+  get: 'Get',
+  post: 'Create',
+  put: 'Replace',
+  patch: 'Update',
+  delete: 'Delete',
+  options: 'Options',
+  head: 'Head',
+  trace: 'Trace',
+} satisfies Record<OpenAPIV3.HttpMethods, string>);
+
+type RouteDictionary = Record<`${OpenAPIV3.HttpMethods} ${string}`, string>;
+
+const devFeatureCustomRoutes: RouteDictionary = Object.freeze({
+  // Security
+  'post /security/subject-tokens': 'CreateSubjectToken',
+});
+
+export const customRoutes: Readonly<RouteDictionary> = Object.freeze({
```

Comment:
Are we going to list every new API in this `customRoutes` object in future development?

Reply:
why? these are exceptions

Reply:
Yeah, after finishing reading the code down below and I realized it myself. No problem for now